templates/components/loaders/python/web.py

```diff
@@ -11,9 +11,17 @@ class WebLoaderConfig(BaseModel):
 
 def get_web_documents(config: WebLoaderConfig):
     from llama_index.readers.web import WholeSiteReader
+    from selenium import webdriver
+    from selenium.webdriver.chrome.options import Options
+
+    options = Options()
```

Comment:
add a `headless` option to `WebLoaderConfig`
default for `create-llama` is set to `false` (it's better for demo videos)
default for `create-tsi` is set to `true`

(don't configure this via UI) - just in the JSON/YAML